portality/dao.py

```diff
@@ -855,34 +862,40 @@ def iterate_unstable(cls, q: dict = None, page_size: int = 1000, limit: int = No
                 else:
                     yield r
 
+            if logger: logger(f"Iterated {counter} records")
+
     @classmethod
     def iterall(cls, page_size=1000, limit=None, **kwargs):
         # TODO: Another candidate for swapping to iterate_pit (or rename iterate_pit to iterate when we're happy)
         return cls.iterate(MatchAllQuery().query(), page_size, limit, **kwargs)
 
     @classmethod
-    def iterall_unstable(cls, page_size=1000, stripe_field="id", striped=False, prefix_generator=None, limit=None, **kwargs):
-        def hex_prefixes(n=3):
+    def iterall_unstable(cls, page_size=1000, stripe_field="id", striped=False, prefix_generator=None, prefix_size=4, limit=None, logger=None, **kwargs):
+        def hex_prefixes(n=4):
```

Comment:
Did we decide that prefix size n=4 is always likely to be overkill? I note we're requesting 3 in most cases